docs/General/Manuals/client-manual.rst

```diff
@@ -61,6 +61,33 @@ The following buttons are available on the :guilabel:`Start Screen`:
 Start New Game
 --------------
 
+Clicking :guilabel:`Start new game` will cause the client to spawn a local ``freeciv21-server`` instance in
```

Comment:
I'm not sure if we want users to know about the server until they want to host their own game. IMHO it's an implementation detail.